src/containers/Main.js

```diff
@@ -28,11 +27,17 @@ export default class Main extends Component {
   }
 
   componentDidMount() {
-    const darkPref = window.matchMedia("(prefers-color-scheme: dark)");
-    this.setState({ isDark: darkPref.matches });
+    if (localStorage.getItem('isDark')==null) {
```

Comment:
Please use strict equality comparison (===) with indentation.

Reply:
I have fixed this ✔
Thanks